New rigellian plague (#4743)
endless-sky/endless-sky: data/human/transport missions.txt

	description "Transport a team of nurses and vaccine supplies to <destination>. Payment will be 100,000 credits."
	deadline
	cargo "vaccine" 1
	passengers 2
	to offer
		random < 100
	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral"
	destination
		distance 3 5
		government "Republic" "Free Worlds" "Syndicate" "Neutral"
	on offer
		conversation
			`The spaceport entry is practically at a standstill today, jammed with a winding line of people standing around irritably. You overhear one person grumble to another "Why'd they have to pick here to do this?" Their companion responds with a grin "You know as well as I do that if they did it somewhere out of the way, no-one would get the vaccination. They keep telling us that it is only a matter of time until it comes back, but how likely is that?"`
			`	As the line moves forward, you can see a banner stating "Red Cross Vaccination Clinic - Free Rigellian Plague Vaccinations." The nurses are surprisingly fast, vaccinating people and giving them a card with information at a fast clip. Before you can even make a decision, you are at the front of the line and a nurse is standing in front of you scanning your ID.`

> 			`	"Good morning Captain! I see your medical records indicate you haven't received the vaccine yet," she pauses, skimming a few lines on her scanner. "Ah, good. You aren't allergic to anything in this vaccine, and haven't had any reactions to medicines in the past. This vaccine is designed to boost your immune system against three variants of the Rigellian Plague. It won't guarantee your immunity, but it will significantly improve your resistance to it. In which arm would you prefer to receive the vaccine?"`

Reviewer: ```suggestion
			`	"Good morning Captain! I see your medical records indicate you haven't received the vaccine yet," she pauses, skimming a few lines on her scanner. "Ah, good. You aren't allergic to anything in this vaccine, and haven't had any reactions to medications in the past. This vaccine is designed to boost your immune system against three variants of the Rigellian Plague. It won't guarantee your immunity, but it will significantly improve your resistance to it. In which arm would you prefer to receive the vaccine?"`
```

			choice
				`	"Sorry, I'm not going that way."`
					decline
				`	"Sure. Here's my ship's berth."`
			`	"Excellent. We will be there in a couple hours."`
			accept
			label risky
			`	"Risky?!" She calls after you. "You're more likely to die flying to the next system than suffer anything from our vaccines." She turns away from you in disgust to treat the next person.`
			fail "The New Rigellian Plague"
	
	on complete
		payment 100000
		event "plague discovered" 2
		dialog
			`The pair of nurses pay you <payment> and quickly unload their precious cargo of vaccines and set up shop next to a busy gateway on <planet>.`
			`	"Thank you for your help, Captain. Hopefully it comes to nothing, but you may have helped save billions."`
	
	on decline
		event "plague discovered" 2
	
	on fail
		event "plague spreading" 2

event "plague discovered"

event "plague spreading"

mission "The New Rigellian Plague Discovered"
	landing
	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral"
	to offer
		has "event: plague discovered"
	on offer
		conversation

> 			`The crowd in the spaceport cantina are worriedly watching a news broadcast where the announcer has just finished listing off a number of planets. "Health authorities have stated that the current symptoms point to the first case of the Rigellian plague in over a century. Officials are currently crediting a recent vaccination push by independent humanitarian groups for preventing this from becoming a full fledged pandemic. We will have more updates after this message from our sponsor."`

Reviewer: ```suggestion
			`The crowd in the spaceport cantina are worriedly watching a news broadcast where the announcer has just finished listing off a number of planets. "Health authorities have stated that the current symptoms point to the first case of the Rigellian plague in over a century. Officials are currently crediting a recent vaccination push by independent humanitarian groups for preventing this from becoming a full-fledged pandemic. We will have more updates after this message from our sponsor."`
```

> 			`	The crowd disperses back to their tables and you can hear a variety of murmurs of people talking about the news and wondering where it might show up next.`

Reviewer: ```suggestion
			`	The crowd disperses back to their tables and you can hear a variety of murmurs of people talking about the news and wondering where the plague might show up next.`
```

			fail

mission "The New Rigellian Plague Spreading"
	landing
	invisible
	source
		government "Republic" "Free Worlds" "Syndicate" "Neutral"
	to offer
		has "event: plague spreading"
	on enter
		conversation

> 			`... After a series of hacking coughs, you finally collapse. Without enough air in you, your limbs begin to numb. In, out, in, out, but it is of no use. You try and drag yourself to your ships panel to send a distress signal but your arms gave out beforehand, leaving you wanting upon the floor. You make a scream, scared and wanting someone, anyone to help you. It is of no use, you could barely get the noise out of you before a wheeze cut it short. Out of oxygen, your vision leaves you. All that remains to you is the hum of the ship and the feel of yourself upon the cold floor. Eventually, the feel of cold goes away. It is neither cold nor warm. As you begin to drift out of consciousness, you think of your life up until now; the journey you've taken. You think of your family back on New Boston waiting for you to make a return. All is quiet.`

Reviewer: ```suggestion
			`... After a series of hacking coughs, you finally collapse. Without enough air in you, your limbs begin to numb. In, out, in, out, but it is of no use. You try and drag yourself to your ship's panel to send a distress signal but your arms gave out beforehand, leaving you wanting upon the floor. You make a scream, scared and wanting someone, anyone to help you. It is of no use, you could barely get the noise out of you before a wheeze cut it short. Out of oxygen, your vision leaves you. All that remains to you is the hum of the ship and the feel of yourself upon the cold floor. Eventually, the feeling of cold goes away. It is neither cold nor warm. As you begin to drift out of consciousness, you think of your life up until now; the journey you've taken. You think of your family back on New Boston waiting for you to make a return. All is quiet.`
```
